src/vopr.zig

```diff
@@ -692,7 +692,7 @@ pub const Simulator = struct {
                     // Check if replica was stuck while repairing headers. Find largest missing
                     // header as we repair headers from high -> low ops.
                     if (replica.journal.find_latest_headers_break_between(
-                        replica.commit_min,
+                        replica.commit_min + 1,
                         @min(replica.op, commit_max),
```

Comment:
will break if commit_min == commit_max?

Reply:
Ah, I thought that since `core_missing_prepare` is supposed to check for cases where we `commit_min < commit_max`, we wouldn't hit that case at all. 🤔

Reply:
Mhmmmmm but `pending` checks for so many more conditions apart from just that. So we _could_ encounter a case where `commit_min == commit_max` but the cluster still hasn't converged in liveness mode...
https://github.com/tigerbeetle/tigerbeetle/blob/cbf7b34d905f7cf10c5ce7307cb754eb31722034/src/vopr.zig#L504

Reply:
Surprising to me that VOPR1000 hasn't yet been able to find a case `core_missing_prepare` breaks due to `commit_min == commit_max`. I went through all of the conditions in `pending` and it _does_ seem to me like `commit_min < commit_max` would be a symptom for all of the conditions that `pending` checks... 

Either way, I revisited that logic in `core_missing_prepare` and clarified it a bit as a part of https://github.com/tigerbeetle/tigerbeetle/pull/2391/commits/741610cef1de769360409ee9e86b6c2482472f6f.